aws/cloudformation/data.yml.erb

```diff
@@ -70,7 +70,7 @@ Resources:
     Properties:
       AllowMajorVersionUpgrade: true
       AllocatedStorage: 250
-      EngineVersion: 3.3.3
+      EngineVersion: 3.4.5
```

Comment:
The CloudFormation Stack update failed because Database Migration Service does not support updating an existing instance to `3.4.5`. We could rename the Logical Resource `TableauSync` in this template to something like `DMSReplicationInstance` and update the one reference in this template to use the new name. That would delete the existing DMS Replication Instance and its tasks, and create a new one with identically configured Tasks. 

The export Ruby script finds the Replication Tasks by using the `describe_replication_tasks` method, which finds all Tasks in an Account & Region and filters them by Tag, so the fact that the Tasks will be deleted and then recreated on a new DMS Replication Instance should not matter. But anytime an engineer says “It should work” that should send chills down your spine. We should probably coordinate this change with the RED team to ensure if/when we carry it out, it’s at a low risk time for them.

Reply:
Thanks, Suresh. What do you think about upgrading to 3.4.4 instead? If this can be updated automatically via CFN, it may also fix the issue in the short term while we figure out the longer term plan.

Reply:
Given that we configure everything in Database Migration Service via CloudFormation, I'm confident that a delete/replace is low risk. I've checked with RED team (@Sam-Hutchins and others) and today is a good day attempt this change. We'll have all weekend to let the next export run, and we have time to revert/fix and re-run the export before they next need refreshed data on Wed if the update is not successful.